docs/pages/components/side-navigation.md

```diff
@@ -325,8 +331,7 @@ The lists in both sections (Main and Utility) should have the `fd-nested-list--c
                         </a>
                     </li>
                     <li class="fd-nested-list__item">
-                        <a class="fd-nested-list__link" 
-                            href="#" aria-controls="EX500L3" aria-haspopup="true">
+                        <a class="fd-nested-list__link">
```

Comment:
Is this meant to have the `href` removed?

Reply:
Could you explain why this attribute has been removed? (I unresolved this conversation)

Reply:
It has been added back in and hrefs from spans removed